appveyor.yml

```diff
@@ -17,6 +17,9 @@ environment:
     - ruby_version: "21" # Older version, but matches Travis-CI
     - ruby_version: "21-x64"
 
+init:
+  - git config --global core.autocrlf true
```

Comment:
@Arcanemagus I took you up on this idea right away. 

https://github.com/skywinder/github-changelog-generator/pull/539